packages/vapor/resources/icons/svg/association-rules-to-group.svg

```diff
@@ -0,0 +1,137 @@
+<svg width="328" height="188" viewBox="0 0 328 188" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I know that for ReactVapor we remove the `width` & `height` attributes from SVGs because it's quite easy to control their size in CSS, for your case I think we could do the same thing but not completely sure.